src/perl/vdotest/VDOTest/VDOClobber.pm

```diff
@@ -0,0 +1,121 @@
+##
+# Test that creating a vdo doesn't clobber a file system or another VDO volume
+#
+# $Id: $
+##
+package VDOTest::VDOClobber;
+
+use strict;
+use warnings FATAL => qw(all);
+use English qw(-no_match_vars);
+use Log::Log4perl;
+
+use Permabit::Assertions qw(
+  assertEvalErrorMatches
+  assertNumArgs
+  assertRegexpMatches
+);
+use Permabit::CommandString::VDOFormat;
+use Permabit::Constants;
+use Permabit::PlatformUtils qw(getDistroInfo);
+
+use base qw(VDOTest);
+
+my $log = Log::Log4perl->get_logger(__PACKAGE__);
+
+##
+# @paramList{getProperties}
+our %PROPERTIES
+  = (
+     # @ple set up a Linear device
```

Comment:
This comment is incorrect. Since these test rely on having an unmanaged device, it might be worth saying something about that. (But it doesn't need an extensive comment.)

Reply:
fixed